MobileApplication/MobileApplication/App.xaml.cs

```diff
@@ -14,6 +14,7 @@ public App()
             InitializeComponent();
 
             //DependencyService.Register<MockDataStore>();
+            Device.SetFlags(new[] { "Expander_Experimental" });
```

Comment:
What does this Flag do?

Reply:
This indicated that the function (Expander, used in floating menu button) is still in experimental state and awaiting feedback. Without it program throws exception.